edi_bank_statement_import/data/data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="edi_exchange_backend" model="edi.backend.type">
+        <field name="name">Bank Statement Backend Type</field>
+        <field name="code">bank_statement_backend_type</field>
+    </record>
+    <record id="edi_exchange_type" model="edi.exchange.type">
```

Comment:
Creating an exchange type by default w/out noupdate is dangerous.

BTW personally I'm thinking about removing default exchange types from the modules I maintain because it can be dangerous too due to their high level of configuration.

I tend to say that you should leave this to the specific setup of the instance and move this to demo data.
Just a recommendation tho. I'd like to hear your feedback and @etobella 's too.